### PR TITLE
Support modification of custom BT tables in deploy_gke.sh

### PR DESCRIPTION
**Pull request by Fructokinase**
This script allows something like this:
./scripts/deploy_gke.sh -p iqss-gdc  -t import_group1_test_2023_03_24_12_00_04 -h f730596

https://gdc.dataverse.org/tools/statvar#sv=Count_MedicalConditionIncident_Contract_COVID

### CODE REVIEW

File: scripts/deploy_gke.sh [hunk 1]
```diff
@@ -87,6 +91,18 @@ if [[ $PROJECT_ID != "" ]]; then
   cd $ROOT/deploy/overlays
   cp custom_kustomization.yaml.tpl kustomization.yaml
   sed -i '' "s/<PROJECT_ID>/$PROJECT_ID/g" kustomization.yaml
+  export PROJECT_ID=$PROJECT_ID
+  yq eval -i '.project = env(PROJECT_ID)' ../base/custom_bigtable_info.yaml
+  yq eval -i '.instance = "dc-graph"' ../base/custom_bigtable_info.yaml
```
Comment (shifucun) [inline scripts/deploy_gke.sh:96]: no need to set instance?

**Review by shifucun — APPROVED** [review]

File: scripts/deploy_gke.sh (hunk 1 above)
Comment (Fructokinase) [reply scripts/deploy_gke.sh:96]: It's an empty file right now. Not sure if the emptiness was intended so it was set dynamically.

Comment (shifucun) [reply scripts/deploy_gke.sh:96]: Ok, then should set the instance.

Comment (Fructokinase) [reply scripts/deploy_gke.sh]: done.

**Pull request merged**